PD Parity and onset age.Rmd

```diff
@@ -815,7 +828,7 @@ A challenge in such observational studies is applying valid control comparisons.
 
 ## Analyses
-Onset age was defined as when Parkinson's was diagnosed, because symptom onset age was collected only in the local survey. Data were fit with simple Bayesian linear models, using the R[@rdevelopmentcoreteam2019_LanguageEnvironmentStatistical] package _brms_. [@burkner2017_BrmsPackageBayesian] Weakly-informative Student _t_ priors were used for the intercept (df = 3, mean = data mean, SD = 10) and standard deviation (df = 3, mean = 0, SD = 10). Based upon the three previous studies, [@frentzel2017_IncreaseReproductiveLife; @haaxma2007_GenderDifferencesParkinson; @yadav2012_CaseControlStudy] we used a proper informative normal prior (mean = ±2.5, SD = 2) for the effect of the number of children (being positive for the dependent variable of age, and negative for year of birth). We ran four chains of 20 000 iterations each.
+Onset age was defined as when Parkinson's was diagnosed, because symptom onset age was collected only in the local survey. Data were fit with simple Bayesian linear models, using the R[@rdevelopmentcoreteam2019_LanguageEnvironmentStatistical] package _brms_. [@burkner2017_BrmsPackageBayesian] Weakly-informative Student _t_ priors were used for the intercept (df = 3, mean = data mean, scale = 10) and standard deviation of the residuals (df = 3, mean = 0, scale = 10) with rejection sampling used to ensure non-negative. The scales were chosen so the prior distributions had moderate probability mass fully covering the range of plausible parameter values. Based upon the three previous studies, [@frentzel2017_IncreaseReproductiveLife; @haaxma2007_GenderDifferencesParkinson; @yadav2012_CaseControlStudy] we used a proper informative normal prior (mean = ±2.5, SD = 2) for the effect of the number of children (being positive for the dependent variable of age, and negative for year of birth). We ran four chains of 20 000 iterations each.
```

Comment:
Hi, just about to incorporate this pull request but just wanted to check on the phrasing of this: "rejection sampling used to ensure non-negative". i.e. should this be expanded to something like "rejection sampling _was_ used to ensure _SOMETHING was/were_ non-negative"?

And could "The scales were chosen..." be "The scale _values_ were chosen…"?

Reply:
Thanks, yes, it wasn't clear that rejection sampling applied to the second prior only. Yes, should be scale values. Updated commit with these changes incorporated.

Reply:
Cheers, will now fetch this into my current development branch.